GPU/GLES/GLES_GPU.cpp

```diff
@@ -56,6 +56,13 @@ enum {
 	FLAG_DIRTYONCHANGE = 64,
 };
 
+static const char *BlacklistGPUNames[] = {
```

Comment:
Rename this "FramebufferFetchBlacklist" to be clear what feature it is blacklisting for.